package.json

```diff
@@ -7,20 +7,29 @@
   "author": "Gruntwork.io <grunty@gruntwork.io>",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "ncc build src/index.ts -o dist"
+    "build": "ncc build src/index.ts -o dist",
+    "test": "jest"
+  },
+  "directories": {
+    "test": "tests"
   },
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/exec": "^1.1.1",
     "@actions/github": "^5.1.1",
     "@actions/tool-cache": "^2.0.1",
     "@octokit/plugin-throttling": "^7.0.0",
+    "@types/jest": "^29.5.4",
     "@vercel/ncc": "^0.36.1",
+    "jest": "^29.6.4",
     "js-yaml": "^4.1.0",
     "os": "^0.1.2",
-    "typescript": "^5.1.6"
+    "ts-jest": "^29.1.1",
+    "typescript": "^5.1.6",
```

Comment:
I _believe_ that typescript should be in the dev dependencies. The devDependencies section represents the ones that aren't made available in the final deployed artifact.

I think you can just remove and re-add using `yarn add -D typescript` to get it into the dev section.

Reply:
Ah good point - fixed.